pygsti/tools/lindbladtools.py

```diff
@@ -238,3 +504,338 @@ def create_lindbladian_term_errorgen(typ, Lm, Ln=None, sparse=False):  # noqa N8
     if sparse:
         lind_errgen = lind_errgen.tocsr()
     return lind_errgen
+
+
+def random_error_generator_rates(num_qubits, errorgen_types=('H', 'S', 'C', 'A'), max_weights=None,
```

Comment:
Should function name include CPTP? i.e. `random_CPTP_error_generator_rates`? I can give you random non-CPTP rates real easy :P